spec/features/deserialization_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.feature "deserialization" do
+  include ActiveJobHelper
+  include RackTestFeaturesHelper
+
+  before do
+    Timecop.travel Time.new(2015, 9, 1, 15, 47).utc
+    Capybara.current_driver = :rack_test
+    VisitMetricsEntry.delete_all
+  end
+
+  after do
+    Timecop.return
+  end
+
+  let(:data) {
+    YAML.load(
+      File.read(
+        File.expand_path('../../fixtures/deserialization_data.yml', __FILE__)
+      )
+    )
+  }
+
```

Comment:
I know this passes: I run Rubocop every time I run rake :-)

However, [we've got Rubocop configured](https://github.com/ministryofjustice/prison-visits/blob/c2d8e43b75fa2d284a9b3ecbb8dded308930618f/.default_rubocop.yml#L12-L14) for [Weirich-style blocks](http://devblog.avdi.org/2011/07/26/the-procedurefunction-block-convention-in-ruby/), i.e. `{...}` for blocks that return a value, and `do ... end` for imperative code.